tests/diff/startup_warnings_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package diff
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+func TestStartupWarnigns(t *testing.T) {
+	t.Parallel()
+
+	// TODO How to enable and disable telemetry when FerretDB is running?
```

Comment:
@AlekSi, and now I need some help understanding how to write dance diff tests for telemetry, as there is no way to set telemetry as true/false/undecided through db commands. So, what's the way to ensure that the telemetry state is set as we expect it?

Reply:
The current way is "wait for FerretDB/FerretDB#1323", I guess :)